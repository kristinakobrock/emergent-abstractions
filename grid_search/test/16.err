  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:02, 23.43it/s] 14%|#############                                                                                | 7/50 [00:00<00:01, 28.29it/s] 20%|##################4                                                                         | 10/50 [00:00<00:01, 25.24it/s] 26%|#######################9                                                                    | 13/50 [00:00<00:01, 26.05it/s] 32%|#############################4                                                              | 16/50 [00:00<00:01, 26.78it/s] 38%|##################################9                                                         | 19/50 [00:00<00:01, 27.57it/s] 44%|########################################4                                                   | 22/50 [00:00<00:01, 26.78it/s] 50%|##############################################                                              | 25/50 [00:00<00:00, 25.09it/s] 56%|###################################################5                                        | 28/50 [00:01<00:00, 26.43it/s] 62%|#########################################################                                   | 31/50 [00:01<00:00, 26.18it/s] 68%|##############################################################5                             | 34/50 [00:01<00:00, 26.00it/s] 74%|####################################################################                        | 37/50 [00:01<00:00, 26.84it/s] 80%|#########################################################################6                  | 40/50 [00:01<00:00, 23.98it/s] 86%|###############################################################################1            | 43/50 [00:01<00:00, 25.33it/s] 94%|######################################################################################4     | 47/50 [00:01<00:00, 26.54it/s]100%|############################################################################################| 50/50 [00:01<00:00, 24.19it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.64it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 31%|############################6                                                                | 4/13 [00:00<00:00, 29.31it/s] 54%|##################################################                                           | 7/13 [00:00<00:00, 25.05it/s] 77%|######################################################################7                     | 10/13 [00:00<00:00, 22.38it/s]100%|############################################################################################| 13/13 [00:00<00:00, 23.15it/s]100%|############################################################################################| 13/13 [00:00<00:00, 23.72it/s]
