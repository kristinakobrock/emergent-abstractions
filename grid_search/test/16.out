# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=128", "--learning_rate=0.0005", "--hidden_size=128", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7679257392883301, "acc": 0.48983123898506165, "length": 3.5340282917022705, "mode": "train", "epoch": 1}
{"loss": 0.6729995012283325, "acc": 0.6021484732627869, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.710185706615448, "acc": 0.6309725642204285, "length": 3.5002388954162598, "mode": "train", "epoch": 2}
{"loss": 0.6779395341873169, "acc": 0.5847656726837158, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6713326573371887, "acc": 0.6180127263069153, "length": 3.4077072143554688, "mode": "train", "epoch": 3}
{"loss": 0.7137837409973145, "acc": 0.43417972326278687, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6718848347663879, "acc": 0.6033861637115479, "length": 3.4174275398254395, "mode": "train", "epoch": 4}
{"loss": 0.6919989585876465, "acc": 0.5306640267372131, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6632989048957825, "acc": 0.6306740045547485, "length": 3.425966501235962, "mode": "train", "epoch": 5}
{"loss": 0.6725788116455078, "acc": 0.6109497547149658, "length": 1.328125, "mode": "test", "epoch": 5}
{"loss": 0.6630498766899109, "acc": 0.6321358680725098, "length": 3.4615440368652344, "mode": "train", "epoch": 6}
{"loss": 0.6702311038970947, "acc": 0.6017853617668152, "length": 3.51953125, "mode": "test", "epoch": 6}
{"loss": 0.6609877943992615, "acc": 0.6292611956596375, "length": 3.469373941421509, "mode": "train", "epoch": 7}
{"loss": 0.6709574460983276, "acc": 0.6002013683319092, "length": 3.9765625, "mode": "test", "epoch": 7}
{"loss": 0.6599093675613403, "acc": 0.6312594413757324, "length": 3.48492169380188, "mode": "train", "epoch": 8}
{"loss": 0.663232684135437, "acc": 0.622851550579071, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.6589821577072144, "acc": 0.632580578327179, "length": 3.4821712970733643, "mode": "train", "epoch": 9}
{"loss": 0.6652324199676514, "acc": 0.6228514313697815, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.658420205116272, "acc": 0.6317391991615295, "length": 3.5378496646881104, "mode": "train", "epoch": 10}
{"loss": 0.6648956537246704, "acc": 0.6234374642372131, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.6570741534233093, "acc": 0.6317201256752014, "length": 3.5548999309539795, "mode": "train", "epoch": 11}
{"loss": 0.6586374640464783, "acc": 0.6251954436302185, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.6556825041770935, "acc": 0.6321004033088684, "length": 3.5886640548706055, "mode": "train", "epoch": 12}
{"loss": 0.6535173654556274, "acc": 0.6271485686302185, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.6519377827644348, "acc": 0.6321812868118286, "length": 3.6368377208709717, "mode": "train", "epoch": 13}
{"loss": 0.6451056003570557, "acc": 0.6306641101837158, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.6475764513015747, "acc": 0.6323360204696655, "length": 3.684182643890381, "mode": "train", "epoch": 14}
{"loss": 0.6378740668296814, "acc": 0.6333985328674316, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.6389139890670776, "acc": 0.6337199807167053, "length": 3.786997079849243, "mode": "train", "epoch": 15}
{"loss": 0.63649582862854, "acc": 0.6378905773162842, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.6290826797485352, "acc": 0.6441220641136169, "length": 3.8584790229797363, "mode": "train", "epoch": 16}
{"loss": 0.6330152750015259, "acc": 0.6431640982627869, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.6174408793449402, "acc": 0.6664254665374756, "length": 3.9028518199920654, "mode": "train", "epoch": 17}
{"loss": 0.631737470626831, "acc": 0.6392578482627869, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.6059146523475647, "acc": 0.675502598285675, "length": 3.926218032836914, "mode": "train", "epoch": 18}
{"loss": 0.6304093599319458, "acc": 0.6363281607627869, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.5972275733947754, "acc": 0.6795513033866882, "length": 3.92889666557312, "mode": "train", "epoch": 19}
{"loss": 0.6227701306343079, "acc": 0.6525390148162842, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.5887579321861267, "acc": 0.6926496028900146, "length": 3.95589542388916, "mode": "train", "epoch": 20}
{"loss": 0.6117807030677795, "acc": 0.6623046398162842, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.5777150988578796, "acc": 0.7029514908790588, "length": 3.9590957164764404, "mode": "train", "epoch": 21}
{"loss": 0.5992742776870728, "acc": 0.6792969107627869, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.5691531896591187, "acc": 0.7098371982574463, "length": 3.9691085815429688, "mode": "train", "epoch": 22}
{"loss": 0.5948030948638916, "acc": 0.6828124523162842, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.5625719428062439, "acc": 0.717719316482544, "length": 3.9755241870880127, "mode": "train", "epoch": 23}
{"loss": 0.5924453139305115, "acc": 0.6929687857627869, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.553217887878418, "acc": 0.7220501899719238, "length": 3.977173089981079, "mode": "train", "epoch": 24}
{"loss": 0.5908990502357483, "acc": 0.6939452886581421, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.5479050874710083, "acc": 0.7235159277915955, "length": 3.9786338806152344, "mode": "train", "epoch": 25}
{"loss": 0.5903216600418091, "acc": 0.6898437142372131, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.5427167415618896, "acc": 0.7275587916374207, "length": 3.986741065979004, "mode": "train", "epoch": 26}
{"loss": 0.581810712814331, "acc": 0.7031250596046448, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.5404787063598633, "acc": 0.7309715747833252, "length": 3.9876556396484375, "mode": "train", "epoch": 27}
{"loss": 0.5806819200515747, "acc": 0.703125, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.5356201529502869, "acc": 0.7349563241004944, "length": 3.987570285797119, "mode": "train", "epoch": 28}
{"loss": 0.5754309892654419, "acc": 0.7078125476837158, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.5277396440505981, "acc": 0.7377461194992065, "length": 3.9920902252197266, "mode": "train", "epoch": 29}
{"loss": 0.5683232545852661, "acc": 0.7109375, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.5272123217582703, "acc": 0.7379432916641235, "length": 3.989016056060791, "mode": "train", "epoch": 30}
{"loss": 0.5725612640380859, "acc": 0.7115234732627869, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.5224002003669739, "acc": 0.7417612075805664, "length": 3.9903676509857178, "mode": "train", "epoch": 31}
{"loss": 0.5704701542854309, "acc": 0.7095703482627869, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.5193616151809692, "acc": 0.7411984205245972, "length": 3.994210720062256, "mode": "train", "epoch": 32}
{"loss": 0.5662957429885864, "acc": 0.71484375, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.5162842273712158, "acc": 0.743415892124176, "length": 3.9949471950531006, "mode": "train", "epoch": 33}
{"loss": 0.5618870258331299, "acc": 0.7173828482627869, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.5154889225959778, "acc": 0.7459046244621277, "length": 3.994502067565918, "mode": "train", "epoch": 34}
{"loss": 0.5651509165763855, "acc": 0.7123046517372131, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.5143150091171265, "acc": 0.7464699149131775, "length": 3.9927480220794678, "mode": "train", "epoch": 35}
{"loss": 0.5571309924125671, "acc": 0.7234373688697815, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.5092759728431702, "acc": 0.7491603493690491, "length": 3.9940543174743652, "mode": "train", "epoch": 36}
{"loss": 0.5485429167747498, "acc": 0.7261718511581421, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.506931722164154, "acc": 0.753377377986908, "length": 3.9972405433654785, "mode": "train", "epoch": 37}
{"loss": 0.5597314238548279, "acc": 0.7208983898162842, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.4977852404117584, "acc": 0.7520723342895508, "length": 3.9949276447296143, "mode": "train", "epoch": 38}
{"loss": 0.5415352582931519, "acc": 0.7296875715255737, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.4944725036621094, "acc": 0.7547283172607422, "length": 3.996222972869873, "mode": "train", "epoch": 39}
{"loss": 0.5526297688484192, "acc": 0.7246094346046448, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.48970475792884827, "acc": 0.7605361938476562, "length": 3.998300313949585, "mode": "train", "epoch": 40}
{"loss": 0.546150267124176, "acc": 0.7341796159744263, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.4843911826610565, "acc": 0.7652915120124817, "length": 3.9964263439178467, "mode": "train", "epoch": 41}
{"loss": 0.5403579473495483, "acc": 0.7310547828674316, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.4783714711666107, "acc": 0.7712913155555725, "length": 3.9967222213745117, "mode": "train", "epoch": 42}
{"loss": 0.5397282242774963, "acc": 0.7369140982627869, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.4744355380535126, "acc": 0.7751744389533997, "length": 3.997375249862671, "mode": "train", "epoch": 43}
{"loss": 0.5394042134284973, "acc": 0.7357421517372131, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.4682736396789551, "acc": 0.7797378897666931, "length": 3.994664192199707, "mode": "train", "epoch": 44}
{"loss": 0.5398733615875244, "acc": 0.7441405653953552, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.46631258726119995, "acc": 0.7823925614356995, "length": 3.9954144954681396, "mode": "train", "epoch": 45}
{"loss": 0.5359039306640625, "acc": 0.7435546517372131, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.46522262692451477, "acc": 0.7848143577575684, "length": 3.9955317974090576, "mode": "train", "epoch": 46}
{"loss": 0.5404210686683655, "acc": 0.7443360090255737, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.459328830242157, "acc": 0.7868852615356445, "length": 3.995084285736084, "mode": "train", "epoch": 47}
{"loss": 0.5259014964103699, "acc": 0.7521483898162842, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.45446035265922546, "acc": 0.7907440066337585, "length": 3.9983913898468018, "mode": "train", "epoch": 48}
{"loss": 0.5265954732894897, "acc": 0.7533202767372131, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.4525465965270996, "acc": 0.792116641998291, "length": 3.99043607711792, "mode": "train", "epoch": 49}
{"loss": 0.5286651253700256, "acc": 0.7570312023162842, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.4463748335838318, "acc": 0.7996322512626648, "length": 3.9929842948913574, "mode": "train", "epoch": 50}
{"loss": 0.5236914157867432, "acc": 0.7535156607627869, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.44466647505760193, "acc": 0.7983036637306213, "length": 3.997654914855957, "mode": "train", "epoch": 51}
{"loss": 0.5258578658103943, "acc": 0.7562499642372131, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.4391826093196869, "acc": 0.8037789463996887, "length": 3.9966676235198975, "mode": "train", "epoch": 52}
{"loss": 0.5153254270553589, "acc": 0.7634764909744263, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.4331898093223572, "acc": 0.8059871792793274, "length": 3.9970390796661377, "mode": "train", "epoch": 53}
{"loss": 0.5201618075370789, "acc": 0.7623047232627869, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.430029958486557, "acc": 0.8074176907539368, "length": 3.987621784210205, "mode": "train", "epoch": 54}
{"loss": 0.5226708650588989, "acc": 0.7632812857627869, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.4286594092845917, "acc": 0.8068543076515198, "length": 3.9925899505615234, "mode": "train", "epoch": 55}
{"loss": 0.517084002494812, "acc": 0.7621093392372131, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.42681631445884705, "acc": 0.8090922832489014, "length": 3.9922962188720703, "mode": "train", "epoch": 56}
{"loss": 0.5156822204589844, "acc": 0.7605467438697815, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.42166802287101746, "acc": 0.8103265166282654, "length": 3.996495008468628, "mode": "train", "epoch": 57}
{"loss": 0.5169677734375, "acc": 0.7603515982627869, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.425353467464447, "acc": 0.8108710646629333, "length": 3.9944472312927246, "mode": "train", "epoch": 58}
{"loss": 0.5217726230621338, "acc": 0.7591796517372131, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.41905826330184937, "acc": 0.8136017322540283, "length": 3.995917320251465, "mode": "train", "epoch": 59}
{"loss": 0.5254342555999756, "acc": 0.7632812857627869, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.4191356599330902, "acc": 0.8147870302200317, "length": 3.995945453643799, "mode": "train", "epoch": 60}
{"loss": 0.5212838053703308, "acc": 0.7566406726837158, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7157813310623169
